sympy/physics/control/tests/test_lti.py

```diff
@@ -1043,6 +1043,36 @@ def test_Feedback_as_TransferFunction():
     assert tf3 == TransferFunction(Series(tf1, fd1), tf2, s)
     assert tf3.num == tf1*fd1
 
+def test_issue_26161():
+    # Issue https://github.com/sympy/sympy/issues/26161
+    Ib, Is, m, h, l2, l1 = symbols('I_b, I_s, m, h, l2, l1',
+                                            real=True, nonnegative=True)
+    KD, KP, v = symbols('K_D, K_P, v', real=True)
+
+    tau1_sq = (Ib + m * h ** 2) / m / g / h
+    tau2 = l2 / v
+    tau3 = v / (l1 + l2)
+    K = v ** 2 / g / (l1 + l2)
+
+    Gtheta = TransferFunction(-K * (tau2 * s + 1), tau1_sq * s ** 2 - 1, s)
+    Gdelta = TransferFunction(1, Is * s ** 2 + c * s, s)
+    Gpsi = TransferFunction(1, tau3 * s, s)
+    Dcont = TransferFunction(KD * s, 1, s)
+    PIcont = TransferFunction(KP, s, s)
+    Gunity = TransferFunction(1, 1, s)
+
+    Ginner = Feedback(Dcont * Gdelta, Gtheta)
+    Gouter = Feedback(PIcont * Ginner * Gpsi, Gunity)
+    assert Gouter == Feedback(Series(PIcont, Series(Ginner, Gpsi)), Gunity)
+    assert Gouter.num == Series(PIcont, Series(Ginner, Gpsi))
+    assert Gouter.den == Parallel(Gunity, Series(Gunity, Series(PIcont, Series(Ginner, Gpsi))))
+    assert Gouter.doit() == \
+           TransferFunction(KD*KP*g*s**3*v**2*(l1 + l2)*(Is*s**2 + c*s)**2*(-g*h*m+s**2*(Ib + h**2*m))* \
+            (-KD*g*h*m*s*v**2*(l2*s + v)+g*v*(l1 + l2)*(Is*s**2+c*s)*(-g*h*m+s**2*(Ib + h**2*m))),
+            s**2*v*(Is*s**2 + c*s)*(-KD*g*h*m*s*v**2*(l2*s + v) + g*v*(l1 + l2)*(Is*s**2 + c*s)*(-g*h*m + s**2*(Ib + \
+            h**2*m)))*(KD*KP*g*s*v*(l1 + l2)**2*(Is*s**2 + c*s)*(-g*h*m+s**2*(Ib + h**2*m))+s**2*v*(Is*s**2 + c*s)* \
+            (-KD*g*h*m*s*v**2* (l2*s + v)+g*v*(l1 + l2)*(Is*s**2 + c*s)*(-g*h*m+s**2*(Ib + h**2*m))))/(l1 + l2), s)
```

Comment:
Equality comparison of long expression is fragile. Best to substract, simiplify and equate to zero.

Reply:
Also, worth checking this is in fact the correct result!

Reply:
I agree Sir, I've reduced the expression

Reply:
Sorry I think you keep misunderstanding me.

1. Yes it is correct to write the full known correct result in the test, like you have done. So keep that.
2. You should do `(known - calculated).simplify == 0` for the test to ensure a robust symbolic test.

Reply:
Thank you, sir. I apologize for any mistakes in my recent contributions to the control. As I recently started contributing, I'm still learning new things, and I appreciate your guidance in helping me improve.

Reply:
There is no need to apologize, nor to call me sir. Your mistakes are fine and you are very close to having this merged.